### ISSUE TITLE
[Bug]: Freetube opens in fullscreen window with no taskbar or minimise/close option

### ISSUE THREAD

**Issue by bagajohny** [FreeTubeApp/FreeTube#4385]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

When I open Freetube it opens in fullscreen window (i dont know how else to describe it) as you can see in the screenshot. The taskbar or the top bar which has option to minimise/close window is not visible at all. I have no clue as to why this is happening. I am using the latest version of Freetube portable app on Windows 10.  The only way to close it is to press Windows key on the keyboard and then close it from the taskbar.

I am able to play videos and browse everything without any issue. I can even toggle video from fullscreen mode to normal mode as usual. Its just that the taskbar and the top bar is not visible. It started out of the blue since last couple of days.

![image](https://github.com/FreeTubeApp/FreeTube/assets/101859073/8637d43c-f18d-4b80-b7a8-96e9642e4524)

### Expected Behavior

The taskbar and the top bar should be visible when the app is open and video is not playing in fullscreen mode.

### Issue Labels

visual bug

### FreeTube Version

v0.19.1-win-x64-portable

### Operating System Version

Windows 10

### Installation Method

Portable

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
Press F11.

FreeTube saves the window position and state, so if you close FreeTube with a window in a certain position or in full screen it will be restored to that when you reopen FreeTube.

TL;DR: if you don't want FreeTube to open in full screen, make sure you exit full screen mode before closing FreeTube.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of https://github.com/FreeTubeApp/FreeTube/issues/3220
